pkgs/development/node-packages/package-tests/postcss-cli.nix

```diff
@@ -0,0 +1,21 @@
+{ runCommand, postcss-cli }:
+
+let
+  inherit (postcss-cli) packageName version;
+in
+
+runCommand "${packageName}-tests" { meta.timeout = 3; }
+  ''
+    # get version of installed program and compare with package version
+    claimed_version="$(${postcss-cli}/bin/postcss --version)"
+    if [[ "$claimed_version" != "${version}" ]]; then
+      echo "Error: program version does not match package version ($claimed_version != ${version})"
+      exit 1
+    fi
+
+    # run basic help command
+    ${postcss-cli}/bin/postcss --help > /dev/null
+
```

Comment:
I'd like to see a tiny amount of css being processed, just to exercise the essential code paths and see that the libraries integrate correctly.

Reply:
AFAICT, postcss-cli has no built-in functionality to process CSS, everything seems to be done via plugins. I was planning to add autoprefixer in a different PR. If you like, I can make that PR and when it is accepted, change this PR to test postcss-cli with autoprefixer.

Reply:
Sounds good to me.

Reply:
I made the other PR.

Reply:
The other PR has been accepted and I updated this PR with a simple autoprefixer test.